scripts/pi-hole/php/teleporter.php

```diff
@@ -151,6 +153,7 @@ function archive_restore_table($file, $table, $flush=false)
 	return $num;
 }
 
+=======
```

Comment:
This prevents the code from working at all due to a syntax error. Remove this line (left over from a merge conflict resolution).